components/user-mgt/org.wso2.carbon.user.mgt/src/main/java/org/wso2/carbon/user/mgt/listeners/UserManagementAuditLogger.java

```diff
@@ -430,4 +498,36 @@ private void addContextualAuditParams(JSONObject jsonObject) {
         jsonObject.put(USER_NAME_KEY, MDC.get(USER_NAME_QUERY_KEY));
         jsonObject.put(SERVICE_PROVIDER_KEY, MDC.get(SERVICE_PROVIDER_QUERY_KEY));
     }
+
+    /**
+     * Set claims in audit logs based on the masking config.
```

Comment:
the description and method name are confusing as doesn't have any values set for audit logs.